### ISSUE TITLE
ShyLU install broken

### ISSUE THREAD

**Issue by VictorEijkhout** [trilinos/Trilinos#11421]
The DealII install reports:

```
CMake Error at /work2/00434/eijkhout/trilinos/installation-13-0-1-frontera-oneapi-impi/lib/cmake/ShyLU/ShyLUConfig.cmake:156 (INCLUDE):
  INCLUDE could not find requested file:

    /work2/00434/eijkhout/trilinos/installation-13-0-1-frontera-oneapi-impi/lib/cmake/ShyLU/../ShyLU_Node/ShyLU_NodeConfig.cmake
Call Stack (most recent call first):
  /work2/00434/eijkhout/trilinos/installation-13-0-1-frontera-oneapi-impi/lib/cmake/Trilinos/TrilinosConfig.cmake:130 (INCLUDE)
  cmake/macros/macro_find_package.cmake:27 (_FIND_PACKAGE)
  cmake/modules/FindTRILINOS.cmake:38 (FIND_PACKAGE)
  cmake/macros/macro_find_package.cmake:27 (_FIND_PACKAGE)
  cmake/configure/configure_20_trilinos.cmake:23 (FIND_PACKAGE)
  /work2/00434/eijkhout/dealii/build-9.4.1-frontera-oneapi-impi/CMakeFiles/CMakeTmp/evaluate_expression.tmp:1 (FEATURE_TRILINOS_FIND_EXTERNAL)
  cmake/macros/macro_evaluate_expression.cmake:30 (INCLUDE)
  cmake/macros/macro_configure_feature.cmake:237 (EVALUATE_EXPRESSION)
  cmake/configure/configure_20_trilinos.cmake:428 (CONFIGURE_FEATURE)
  cmake/macros/macro_verbose_include.cmake:19 (INCLUDE)
  CMakeLists.txt:130 (VERBOSE_INCLUDE)


CMake Warning at /work2/00434/eijkhout/trilinos/installation-13-0-1-frontera-oneapi-impi/lib/cmake/Trilinos/TrilinosConfig.cmake:140 (MESSAGE):
  Component "ShyLU_Node" NOT found.
Call Stack (most recent call first):
  cmake/macros/macro_find_package.cmake:27 (_FIND_PACKAGE)
  cmake/modules/FindTRILINOS.cmake:38 (FIND_PACKAGE)
  cmake/macros/macro_find_package.cmake:27 (_FIND_PACKAGE)
  cmake/configure/configure_20_trilinos.cmake:23 (FIND_PACKAGE)
  /work2/00434/eijkhout/dealii/build-9.4.1-frontera-oneapi-impi/CMakeFiles/CMakeTmp/evaluate_expression.tmp:1 (FEATURE_TRILINOS_FIND_EXTERNAL)
  cmake/macros/macro_evaluate_expression.cmake:30 (INCLUDE)
  cmake/macros/macro_configure_feature.cmake:237 (EVALUATE_EXPRESSION)
  cmake/configure/configure_20_trilinos.cmake:428 (CONFIGURE_FEATURE)
  cmake/macros/macro_verbose_include.cmake:19 (INCLUDE)
  CMakeLists.txt:130 (VERBOSE_INCLUDE)
```

(I've suggested before that you should add DealII to your regression tests. It seems good at finding errors in the trilinos installation.)
[trilinos.log](https://github.com/trilinos/Trilinos/files/10287431/trilinos.log)

**Comment by cgcgcg** [comment]
@trilinos/shylu

**Comment by ndellingwood** [comment]
Hopefully resolved by #12245
